src/prob/pf.jl

```diff
@@ -568,7 +569,7 @@ end
 
 Branch component interface outputs branch primitive Y matrix.
 """
-function _cpf_branch_interface(branch::Dict{String,<:Any}, v_start::Dict{<:Any,<:Any})
+function _cpf_branch_interface(branch::Dict{String,<:Any}, v_start::Dict{<:Any,<:Any}, explicit_neutral::Bool)
     br_size = size(branch["br_r"],1)
     @assert br_size <= 4  "Line impedance matrix should be up to 4x4, but is $(br_size)x$(br_size)"
```

Comment:
I'm thinking, this assert is probably better to remove here - if we keep it it's probably better to do it some place in compute_pf. (we may want to run the 6-wire case again, it may actually pass now?)